src/Raven.Server/Documents/Handlers/ChangesHandler.cs

```diff
@@ -111,6 +111,8 @@ private async Task HandleConnection(WebSocket webSocket, JsonOperationContext co
                     var jsonParserState = new JsonParserState();
                     using (var parser = new UnmanagedJsonParser(context, jsonParserState, debugTag))
                     {
+                        connection.EnableTopologyChangeNotification();
```

Comment:
Can we rename that to 'SendSupportedFeatures' and have a single notification for supported features (more future proof)

```json
{
   "TopologyChange" : true
}
```

Reply:
I like that, yes.